atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1642,9 +1642,18 @@ private void putMetadataForTablesInternal(
     }
 
     private void putMetadataWithoutChangingSettings(final TableReference tableRef, final byte[] meta) {
+        long ts = System.currentTimeMillis();
+
+        Multimap<Cell, Long> oldVersions = getAllTimestamps(
```

Comment:
Is it worth checking if the most recent old version == what we're about to put and then short circuiting, or will that get handled by someone else before we get this far?